th/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: page
+title: Concurrency
+category: advanced
+order: 4
+lang: en
```

Comment:
We need to remove all of the `en` lessons from this PR, unfortunately they will screw up the generated content.